routers/repo/pull.go

```diff
@@ -1162,6 +1162,22 @@ func DownloadPullDiffOrPatch(ctx *context.Context, patch bool) {
 
 	pr := issue.PullRequest
 
+	if pr.HasMerged {
+		diffType := git.RawDiffNormal
+		if patch {
+			diffType = git.RawDiffPatch
+		}
+		if err := git.GetRawDiff(
+			models.RepoPath(ctx.Repo.Owner.Name, ctx.Repo.Repository.Name),
+			pr.MergedCommitID,
+			diffType,
+			ctx.Resp,
```

Comment:
Down below we use `ctx` to fulfill `io.Writer`, so why `ctx.Resp` here? It seems both work for the interface, but should they be consistent or is there a specific reason to use `ctx.Resp`?

Reply:
no this call was copyed from code witch serve commit diff/patch